### DISCUSSION TITLE
My point of view

### DISCUSSION THREAD

**Discussion by JeshuathMHS** [hackatomik/SISCOx256#35] [Chill | Carrera]
Quiero ser honesto y comentarles que no entiendo muchos las clases, recomendaria que fueran un poco más lentos y detallados a la hora de explicar ya que para muchos es algo nuevo.

**Comment by AlejandraMichellx** [comment]
Opino lo mismo que Jesuath, en si entiendo las clases y se en para que usamos GitHub mas no, no entiendo como usarlo y agradecería que nos explicaran mas detallado como usar el sitio, ya que esto no me pasa a mi sino que también a dos que tres mas. @tapilew @JeshuathMHS @JamSarmientoH9

**Comment by quantumquirkxyz** [comment]
Es completamente comprensible lo que mencionan, @JeshuathMHS y @AlejandraMichellx. GitHub y las herramientas que estamos utilizando pueden ser nuevas y algo confusas al principio, sobre todo cuando no se ha trabajado con ellas antes. La sugerencia de ir más despacio y explicar las cosas de manera más detallada es muy válida, y creo que puede beneficiar a todos los que están en la misma situación.

Lo que podemos hacer es tomarnos más tiempo para explicar cada paso de forma clara, asegurándonos de que todos entiendan cómo funciona GitHub y sus características básicas antes de avanzar a temas más complejos. Podríamos incluso dedicar más tiempo a ejemplos prácticos y hacer pausas para resolver cualquier duda. El objetivo es que todos se sientan cómodos usando la plataforma, ya que es una herramienta fundamental, no solo para este curso, sino también en el mundo laboral.

Entender bien GitHub desde el principio es clave para poder avanzar con confianza, y me parece una excelente idea que también consideremos hacer una pequeña revisión de lo que ya se ha enseñado, de modo que todos puedan ponerse al día. Al final del día, todos estamos aquí para aprender, y tomarnos el tiempo necesario es lo mejor para asegurarnos de que nadie se quede atrás.

**Reply by quantumquirkxyz** [reply]
[Lecture Note {Git y GitHub}.pdf](https://github.com/user-attachments/files/16914201/Lecture.Note.Git.y.GitHub.pdf)

**Comment by tapilew** [comment]
Recomiendo además del recurso compartido por Jhuomar @Quantumquirk31 el README de este mismo repositorio. Al entrar en el enlace es la primera información que se ve.

https://github.com/hackatomik/SISCOx256

Ahí he incluído todos los recursos necesarios para dominar estos temas. Sugerencia verlo en orden y detenidamente, con calma. Cualquier detalle que no se entienda para eso está este foro de discusiones y nuestros chat personales.
